src/main/kotlin/io/icure/md/client/mappers/Patient.kt

```diff
@@ -139,8 +139,73 @@ fun Patient.toPatientDto() = PatientDto(
 )
 
 fun Patient.DeactivationReason.toDeactivationReason() = PatientDto.DeactivationReason.valueOf(this.name)
+fun io.icure.kraken.client.models.PatientDto.DeactivationReason.toDecryptedDeactivationReason() =
+    PatientDto.DeactivationReason.valueOf(this.name)
+
 fun Patient.Gender.toGender() = PatientDto.Gender.valueOf(this.name)
+fun io.icure.kraken.client.models.PatientDto.Gender.toGender() = PatientDto.Gender.valueOf(this.name)
 fun Patient.BirthSex.toBirthSex() = PatientDto.BirthSex.valueOf(this.name)
+fun io.icure.kraken.client.models.PatientDto.BirthSex.toBirthSex() = PatientDto.BirthSex.valueOf(this.name)
 fun Patient.PersonalStatus.toPersonalStatus() = PatientDto.PersonalStatus.valueOf(this.name)
+fun io.icure.kraken.client.models.PatientDto.PersonalStatus.toPersonalStatus() =
+    PatientDto.PersonalStatus.valueOf(this.name)
 
 fun Patient.Gender.toDbGender() = io.icure.kraken.client.models.PatientDto.Gender.valueOf(this.name)
+
+fun io.icure.kraken.client.models.PatientDto.toDecryptedPatientDto(): PatientDto = PatientDto(
```

Comment:
Use PatientMapper of icure-reactive-kotlin-client (in extendedapis/PatientApi) instead of rewriting code